src/experiments/logins/schema.json

```diff
@@ -199,6 +199,32 @@
       }
     ],
     "functions": [
+      {
+        "name": "getLoginSavingEnabled",
+        "type": "function",
+        "description": "For the given origin, gets a boolean flag which determines whether login saving is enabled",
+        "async": true,
+        "parameters": [{
+          "name": "aHost",
+          "type": "string",
+          "description": "The origin for which to check the status of login saving"
```

Comment:
Just use the description from [the IDL](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Reference/Interface/nsILoginManager#getLoginSavingEnabled()): `The hostname to check. This argument should be in the origin URL format, with no pathname. For example: "https://www.site.com".`

Reply:
Actually, looking at the schema docs, I haven't been super strict about reusing the IDL descriptions. This could be fine as is (but `aHost` does bug me ;-P).